config.template

```diff
@@ -1,88 +1,109 @@
 [case]
-# ACME case configuration
-casename = 20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00
-native_res = ne30
-short_term_archive = 0
-ref_casename_v0 = B1850C5_ne30_v0.4
+
+# casename is a name that identifies the simulation being analyzed.
+casename = casename
+# ref_casename is the name of a reference run to compare agains (or None
+# to turn off comparison with a reference, e.g. if no reference case is
+# available)
+ref_casename = None
```

Comment:
@xylar: sorry for the delay, I am finally taking a look at this now. 
As far as ref_casename goes, I added a '_v0' because we want to keep the reference casename variable different from ACME v0 and v1. We already have a ref_casename to be used to represent a version of ACME greater than 0 (ie, strictly MPAS), so I used ref_casename_v0 for an ACME v0 case (POP based, hence treated as pre-processed data). Mind you, there aren't many v0 simulations we will want to compare to. Here is an exhaustive list:
1) ACME v0 low-res (1deg atm/land, 1deg POP/CICE)
2) ACME v0 high-res, pre-industrial (0.25deg atm/land, 0.1deg POP/CICE)
3) ACME v0 high-res, present-day transients (same resolution as 2)

So, to summarize, we want to keep ref_casename_v0 separate from ref_casename (we are not using the latter, in the current analysis).